deepinfra/types/automatic_speech_recognition/request.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
```

Comment:
I think for all dataclasses pass `kw_only=True`, there are way too many fields to pass by position.